frontend/src/stylesheets/Authentication.scss

```diff
@@ -27,7 +27,6 @@ $auth-dark: #d9535d;
 }
 
 .Auth-company-name {
-  text-align: center;
```

Comment:
To jeśli się nie mylę nie powinno być usunięte.

Reply:
to po prostu przywrócę, usunąłem w sumie niechcący